packages/core/src/v3/zodNamespace.ts

```
@@ -5,12 +5,19 @@ import {
   ZodSocketMessageHandlers,
+  GetSocketMessagesWithCallback,
 } from "./zodSocket.js";
 // @ts-ignore
 import type { DefaultEventsMap, EventsMap } from "socket.io/dist/typed-events";
 import { z } from "zod";
 import { SimpleStructuredLogger, StructuredLogger } from "./utils/structuredLogger.js";
 
+type AssertNoCallbackSchemas<T extends ZodSocketMessageCatalogSchema> = [
+  GetSocketMessagesWithCallback<T>,
+] extends [never]
+  ? {}
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)